src/packages/cli/helpers/build.js

```diff
@@ -23,7 +23,7 @@ async function main() {
       target: 'node10',
       outfile: 'build/index.js',
       entryPoints: ['src/bin.ts'],
-      external: ['@prisma/engines'],
+      external: ['@prisma/engines', '_http_common'],
```

Comment:
What is this about @millsp?

Reply:
`undidci` uses a private Node.js module called `_http_common`

Reply:
https://github.com/nodejs/undici/blob/v3.3.4/lib/node/http-parser.js

Reply:
Seems to be removed in their `v4.0.0` release

Reply:
```
// TODO: This is not really allowed by Node but it works for now.
const common = require('_http_common')
```

Interesting 😄